sparkly/utils.py

```diff
@@ -93,6 +181,45 @@ def kafka_get_topics_offsets(host, topic, port=9092):
     return offsets
 
 
+class lru_cache(object):
```

Comment:
Honestly I've never used the "LRU" part of this cache. I'd imagine that when targeted towards dataframes probably you'd either keep it unlimited or very small like `maxsize=1` or `2`.